src/main/java/org/logstash/beats/Runner.java

```diff
@@ -0,0 +1,46 @@
+package org.logstash.beats;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.logstash.netty.PrivateKeyConverter;
+import org.logstash.netty.SslSimpleBuilder;
+
+import java.io.FileInputStream;
+
+
+public class Runner {
+    private static final int DEFAULT_PORT = 5044;
+    public static Logger logger = LogManager.getLogger(Runner.class.getName());
+
+
+    static public void main(String[] args) throws Exception {
+        logger.info("Starting Beats Bulk");
+
+        // Check for leaks.
+        // ResourceLeakDetector.setLevel(ResourceLeakDetector.Level.PARANOID);
+
+        Server server = new Server(DEFAULT_PORT);
+
+        if(args.length > 0 && args[0].equals("ssl")) {
+            logger.debug("Using SSL");
+
+            String sslCertificate = "/Users/ph/es/certificates/certificate.crt";
```

Comment:
Well I needed to use the certificate outside of the java code for filebeat, I could have written it to disk. will change that.